packages/terra-date-picker/src/react-datepicker/calendar.jsx

```diff
@@ -363,16 +369,16 @@ export default class Calendar extends React.Component {
 
   increaseMonth = () => {
     this.setState({
-      date: addMonths(cloneDate(this.state.date), 1)
+      date: addMonths(cloneDate(this.state.date), 1).startOf('month')
     }, () => this.handleMonthChange(this.state.date))
-    this.props.setPreSelection(addMonths(cloneDate(this.state.date), 1));
+    this.props.setPreSelection(addMonths(cloneDate(this.state.date), 1).startOf('month'));
```

Comment:
`increaseMonth` is called when the next Month button is clicked. Whenever the user interacts using the mouse (instead of keyboard) the focus on the preselected day should not display. See this [comment](https://github.com/cerner/terra-framework/pull/994#discussion_r386692700). So should we update `this.state.calendarIsKeyboardFocused` to false or at least `this.props.calendarOpenedViaKeyboard` should be false so that the preselected day should not get the focus?

Reply:
Changed e4930e6